src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,11 +1,64 @@
 package core.basesyntax;
 
-/**
- * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн Builder.
- * Добавьте не менее пяти полей в класс Plain. Внести все необходимые изминения в класс Plane
- * которые могут потребоваться для реализации паттерна Builder.</p>
- */
-
 public class Plane {
 
+    private String brand;
+    private int model;
+    private int power;
+    private String color;
+    private int wheelsDiameter;
+
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
```suggestion
    private Plane(PlaneBuilder planeBuilder) {
```
explain why in comment below

Reply:
This should be public, becarse this allows passing different types of builders to the director.

Reply:
I changed it to private) so please explain me  why it should be this way?

Reply:
Oh, because we don't want to change fields of the Object throw Builder